data/registry/otel-odddotnet.yml

```diff
@@ -0,0 +1,29 @@
+# cSpell:ignore Kenna odddotnet
+title: OddDotNet OTel Collector Test Harness
```

Comment:
```suggestion
title: OddDotNet: Test Harness for OpenTelemetry Collector
```

Reply:
I like this change, but now it has me wondering if I should remove "Collector" altogether. It's really a test harness for OpenTelemetry in general, so "OddDotNet: Test Harness for OpenTelemetry" might make more sense. I'll make that change, and if that doesn't work for you I'll change it to what you suggested above tomorrow. Thanks!